weed/server/filer_server_handlers_read.go

```diff
@@ -142,6 +142,9 @@ func (fs *FilerServer) GetOrHeadHandler(w http.ResponseWriter, r *http.Request)
 
 	if rangeReq := r.Header.Get("Range"); rangeReq == "" {
 		ext := filepath.Ext(filename)
+		if len(ext) > 0 {
```

Comment:
No need to check len.

Reply:
Sorry, i'm not a Go programmer... i've only borrowed from this: https://github.com/chrislusf/seaweedfs/blob/570d2eccec32d3ec9dfc63ec1688bff27eb7d9e1/weed/server/volume_server_handlers_read.go#L268

Reply:
thanks! Merged. I hope this can help you to get started with Go.